packages/wrangler/src/deploy/index.ts

```diff
@@ -250,17 +251,23 @@ export async function deployHandler(
 		);
 	}
 
-	if ((args.assets || config.assets) && (args.site || config.site)) {
+	if (
+		(args.legacyAssets || config.legacy_assets) &&
```

Comment:
should this not be `args.legacyAssets || config.legacy_assets || config.assets`? Technically `wrangler.toml` still supports `assets` no?

Reply:
nvm. `rawConfig.assets` and `rawConfig.legacy_assets` are merged and normalized into `config.legacy_assets`. Nice one!